### DISCUSSION TITLE
SABnzbd:  file/folder permissions

### DISCUSSION THREAD

**Discussion by memilanuk** [tteck/Proxmox#1767] [Help Needed]
Hello there,

I have sabnzbd installed using the helper script, and am able to access the web UI using tailscale (also set up using the helper-scripts).  I used the approach [mentioned here](https://github.com/tteck/Proxmox/discussions/1416#discussioncomment-5895982) for qbittorrent, and again for the sabnzbd container.  Whereas in the qbit LXC I used `mp0: /mnt/pve/torrents,mp=/data`, in the sab LXC I used `mp0: /mnt/pve/usenet,mp=/data`.  I did two separate CIFS mounts in the Proxmox Datacenter, hoping to prevent either application being able to see or mess with the data directory for the other.  

I can 'see'  the directory contents from the console 'inside' the containers i.e. 'ls -al /data' shows the folders it should.  Permissions look to be 755, user and group of 'nobody'.  The qbittorrent container had no issues accepting its folder, but the sabnzbd container is refusing to validate the paths for the complete/incomplete downloads - '/data/completed' and '/data/incomplete' respectively - giving an error message of "complete_dir directory: /data/completed error accessing".

The research I've done thus far seems to indicate that this is a permissions issue on the server side... but I'm not sure how.  The SMB/CIFS mount is to my Synology NAS, and the share is mounted under my username / password so it should have more than adequate permissions.  I've tried `sudo chmod -R 777 completed` on the server side, but it's still showing as 755/nobody/nogroup on the container side, and while I can 'see' the files I can't do anything with them - like change permissions, etc.  Is there a way to fudge this at the mount point?

Any ideas or suggestions?

**Comment by memilanuk** [comment]
Okay... figured out it's an issue with using un-privileged containers in Proxmox.  They can 'see' the SMB/CIFS share created in the Datacenter -> Storage, but the uid/gid maps to nobody/nobody in the container unless you go through some gyrations to make it work.

https://forum.proxmox.com/threads/tutorial-unprivileged-lxcs-mount-cifs-shares.101795/

**Reply by pokebob** [reply]
Holy spent way to long messing with permissions THANK YOU
